Classes/EventListener/AsyncEventListenerInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace Neos\Cqrs\EventListener;
+
+/*
+ * This file is part of the Neos.Cqrs package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+/**
+ * Marker interface for asynchronous event listeners
```

Comment:
What about adding another sentence:

"Asynchronous event listeners are not triggered directly when new events are added to the system, but only when calling projection:catchup".